.travis.yml

```diff
@@ -27,6 +27,8 @@ virtualenv:
   system_site_packages: true
 
 install:
+  - git clone https://github.com/OCA/reporting-engine ${HOME}/reporting-engine -b ${VERSION}
+  - git clone https://github.com/OCA/web ${HOME}/web -b ${VERSION}
   - git clone https://github.com/oca/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
```

Comment:
you can add ` --depth 1` -> ` git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools`